### ISSUE TITLE
Postman window app dark mode font color need some changes

### ISSUE THREAD

**Issue by developer-shajib** [a85/POSTMan-Chrome-Extension#11417]
### Is there an existing request for this feature?

- [X] I have searched the existing issues for this feature request and I know that duplicates will be closed

### Is your feature request related to a problem?

Postman windows version app dark mode some font color is so poor white..it's very problem when we use..please kindly solve it and font color make fully white.. 

### Describe the solution you'd like

if you change the postman windows version dark mode font color to fully white it's very helpful..

**Comment by pan0ptic0n** [comment]
Agree. The dark mode is awful and blurry on certain monitor settings.
